mailbox/maildir/src/test/java/org/apache/james/mailbox/maildir/OsDetector.java

```diff
@@ -29,8 +30,10 @@
      * @return windows
      */
     public static boolean isWindows() {
-        String os = System.getProperty("os.name").toLowerCase(Locale.US);
-        return (os.contains("win"));
+        return Optional.ofNullable(System.getProperty("os.name"))
+            .orElse("")
+            .toLowerCase(Locale.US)
+            .contains("win");
```

Comment:
No tests?

(we could extract a submethod taking a `Supplier<String> osSupplier` and test it)

Reply:
it is not used